### PR TITLE
Two sum function

### PR DESCRIPTION
**Pull request by elainesmith**
- Created Two Sum function
- Added unit tests for Two Sum

### CODE REVIEW

File: functions/two_sum.py [hunk 1]
```diff
@@ -1,2 +1,6 @@
 def twoSum(n, t):
```
Comment (elainesmith) [inline functions/two_sum.py:1]: I would recommend using more descriptive words for the arguments instead of n & t!

File: tests/two_sum_test.py [hunk 2]
```diff
@@ -1,2 +1,37 @@
 import pytest
 from functions.two_sum import twoSum
+
```
Comment (elainesmith) [inline tests/two_sum_test.py:3]: Great start! I suggest thinking about possible edge cases and writing tests for those as well!

File: tests/two_sum_test.py [hunk 3]
```diff
@@ -1,2 +1,37 @@
...
+def test_returns_0_1_for_3_3_with_target_6():
+    # Arrange
+    n = [3, 3]
+    t = 6
+
+    # Act
+    answer = twoSum(n, t)
+
+    # Assert
+    assert answer == [0, 1]
```
Comment (elainesmith) [inline tests/two_sum_test.py:37]: It's good practice with Pep8 to leave a blank line at the end of the file.

**Review by elainesmith — COMMENTED** [review]
Overall very nice start! The function runs as intended and passes all tests.
My recommendation for improvement would be to consider possible edge cases now and think about ways to add to your code and test for those edge cases in order to make this function more robust. 
Since there are no bugs I'm going to go ahead and approve this PR. Nicely done!
